src/battle_controller_player.c

```diff
@@ -1889,6 +1889,19 @@ static void HandleChooseActionAfterDma3(u32 battler)
     {
         gBattle_BG0_X = 0;
         gBattle_BG0_Y = DISPLAY_HEIGHT;
+        if (gBattleStruct->aiDelayTimer != 0)
+        {
+            gBattleStruct->aiDelayFrames = gMain.vblankCounter1 - gBattleStruct->aiDelayTimer;
```

Comment:
Thoughts on adding the frames counter to the battle debug menu?

Reply:
The AI one or in more general sense?

Reply:
Was thinking just adding the value to the AI Info page but that can just be a separate PR if desired. Otherwise I think this looks good now

Reply:
Let's do it in a separate PR then